Consul/Consul.csproj

```diff
@@ -31,7 +30,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
-  <Target Name="ILMerge" AfterTargets="AfterBuild" Condition=" '$(TargetFramework)' == 'net461' ">
+  <Target Name="ILMerge" AfterTargets="AfterBuild" Condition=" '$(TargetFramework)' == 'net461' And '$(OS)' == 'Windows_NT'">
```

Comment:
Why the new condition? Also it's going to move to ILRepack, right?

Reply:
`ILMerge` is an application that works only on Windows, but we now build `net461` on non-Windows platforms too. 

I'm not sure whether `ILRepack` works on non-Windows platforms (probably yes). We can make a new PR to switch back to `ILRepack` but I think it should be in a separate PR.

Reply:
Sure